Earthfile

```diff
@@ -530,7 +529,17 @@ build-demo-container:
     COPY +install-python/bin /root/.local/bin
     COPY demo demo
     CMD bash
-    SAVE IMAGE ghcr.io/feldera/demo-container
+
+# TODO: mirrors the Dockerfile. See note above.
+build-kafka-connect-container:
+    FROM debezium/connect:2.3
```

Comment:
For this case, since we are not copying anything from other Earthly stages, you can just do `FROM DOCKERFILE -f deploy/Dockerfile --target kafka-connect` and avoid the duplication entirely. https://docs.earthly.dev/docs/earthfile#from-dockerfile

Reply:
nice, didn't realize this was possible.